src/db/index.ts

```diff
@@ -79,3 +98,31 @@ export async function saveCommit(commitInfo: CommitInfo): Promise<void> {
     Sentry.captureException(err);
   }
 }
+
+export interface CommitWithDiagnoses extends Commit {
+  diagnoses: CommitDiagnosis[];
+}
+
+export async function getTopCommitsWithDiagnoses(
+  afterDate: Date,
+  limitCount = 10
+): Promise<CommitWithDiagnoses[]> {
+  const commits = await getTopCommitsAfterDate(afterDate, limitCount);
```

Comment:
Can you make it to use a SQL JOIN clause instead of manually performing the table join in code?

Reply:
I tried a couple different ways including with a JOIN. The code with doing this in 2 queries is a bit cleaner. I can change it to use a join if you want

Reply:
That's okay, not a big problem. Let's keep this as-is for now.